dotCMS/src/main/java/org/apache/velocity/runtime/RuntimeInstance.java

```diff
@@ -1699,6 +1699,13 @@ public boolean dumpVMNamespace(String namespace)
         return vmFactory.dumpVMNamespace( namespace );
     }
 
+    public void dumpGlobalVMNamespace( )
+    {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the end of previous line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ALeftCurlyBraceEndLineCheck)